cai_causal_graph/time_series_causal_graph.py

```diff
@@ -554,6 +556,7 @@ def extend_graph(
                         destination=extended_graph.get_node(lagged_dest.identifier),
                         edge_type=edge.get_edge_type(),
                         meta=edge.meta,
+                        validate=False,
```

Comment:
Same comment. Are we safe to do this?

Reply:
See above

Reply:
Would just add a comment here so we remember why we are setting it to `False`.